library/Requests/Transport/fsockopen.php

```diff
@@ -174,7 +160,12 @@ public function request($url, $headers = array(), $data = array(), $options = ar
 		}
 		$out .= "\r\n";
 
-		$out .= "User-Agent: {$options['useragent']}\r\n";
+		$lowerHeaders = array_change_key_case($headers);
```

Comment:
`$lower_headers` as snake_case instead please. Also didn't realise this function existed, cool. :)

We'll eventually change this to be a proper [CaseInsensitiveDictionary](https://github.com/rmccue/Requests/blob/master/library/Requests/Utility/CaseInsensitiveDictionary.php), but this works nicely in the meantime.